easybuild/easyconfigs/c/catboost/catboost-1.2-foss-2022a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'catboost'
```

Comment:
please rename to `CatBoost` which seems to be the official software name(see description)

Reply:
done